Automattic/wp-calypso
# OAuth: Remove the code to fetch an OAuth signup URL

Once #17530 is merged we can remove calls to get the OAuth signup URL.

#### Testing instructions
  
1. Run `git checkout update/remove-oauth-signup-link` and start your server
2. Open the [`WPCC` login page](http://calypso.localhost:3000/log-in?client_id=390)
3. Check that there are no requests to /oauth2/signup-url/
  
#### Reviews
